net/vpnbypass/files/vpnbypass.init

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh /etc/rc.common
 
-START=92
+START=99
```

Comment:
So something is very strange going on. I've tried START=94 with simple-adblock and it works pretty well -- it downloads on start, so WAN is settled and ready.

However when I use START=94 in vpnbypass and do ```source /lib/functions/network.sh; network_get_gateway wan_gw wan;``` in start_service I get empty $wan_gw, without which I can't start vpnbypass. At least that happens when the package is pre-installed in the image on first boot.

I've tried: ```c=1; while : ; do source /lib/functions/network.sh; network_get_gateway wan_gw wan; [ $c -ge 5 ] && break || let "c+=1"; [ -n "$wan_gw" ] && break || output "waiting for wan gateway...\n"; sleep 5; done```

And that doesn't help at all, I just get 5 "waiting..." messages in the log.

So, is there something I can do about network_get_gateway or use something else?